tools/PI/DevHome.PI/Controls/AddToolControl.xaml

```diff
@@ -23,26 +23,22 @@
                     <Grid.ColumnDefinitions>
```

Comment:
I noticed this is all in Borders, but the borders don't have a brush or thickness. Do the rows need to be in Borders?